packages/botonic-react/src/components/message/timestamps.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+
+import { WEBCHAT } from '../../constants'
+import { TimestampContainer, TimestampText } from './styles'
+
+export const resolveMessageTimestamps = (
+  getThemeFn,
+  messageTimestampEnabled
+) => {
+  const timestampsFormat = getThemeFn(
+    WEBCHAT.CUSTOM_PROPERTIES.messageTimestampsFormat
+  )
+  const timestampStyle = getThemeFn(
+    WEBCHAT.CUSTOM_PROPERTIES.messageTimestampsStyle
+  )
+  const timestampsEnabled = Boolean(
+    messageTimestampEnabled !== undefined
+      ? messageTimestampEnabled
+      : getThemeFn(
+          WEBCHAT.CUSTOM_PROPERTIES.enableMessageTimestamps,
+          Boolean(timestampStyle) || Boolean(timestampsFormat) || false
+        )
+  )
+
+  const defaultTimestampFormat = {
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+  }
+
+  const getFormattedTimestamp = timestampsFormat
+    ? timestampsFormat()
+    : new Date().toLocaleString(undefined, defaultTimestampFormat)
+
+  return {
+    timestampsEnabled,
+    getFormattedTimestamp,
+    timestampStyle,
+  }
+}
+
+export const MessageTimestamp = ({
+  timestamp,
+  style,
+  isSentByUser,
+  sentBy,
```

Comment:
do we need this to prop? can we not get `isSentByUser` ny using the `sentBy` prop

Reply:
Agree with Aina

Reply:
ok I will remove the isSentByUser